cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -47,6 +45,10 @@ func RelayProphecyClaimToEthereum(
 
 	amount := claim.Amount.BigInt()
 
+	if claim.Symbol == "rowan" {
```

Comment:
We should have already done this earlier on with our EthereumToSifchain or SifchainToEthereum translation, we should be able to remove this code.